<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Antivirus Scanner Pro</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_title_sidebar">
         <property name="text">
          <string>üõ°Ô∏è NOTOVIRUS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_version">
         <property name="text">
          <string>v1.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_home">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>üè† Accueil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_file">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>üìÑ Scan Fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_folder">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>üìÅ Scan Dossier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ip">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>üåê Scan IP</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbl_footer">
         <property name="text">
          <string>¬© 2025 Antivirus Pro</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked_pages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_home">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lbl_home_icon">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>üõ°Ô∏è</string>
          </property>
          <property name="alignment">
          <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_home_title">
          <property name="text">
           <string>Bienvenue dans l'Antivirus Scanner</string>
          </property>
          <property name="alignment">
          <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_home_subtitle">
          <property name="text">
           <string>Protection avanc√©e pour vos fichiers, dossiers et adresses IP</string>
          </property>
          <property name="alignment">
          <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_file">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_file_title">
          <property name="text">
           <string>üìÑ Scan de Fichier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_file_selection">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_select_file">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>üìÇ Choisir un fichier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_file_path">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Chemin du fichier...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_scan_file">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="text">
              <string>üîç Lancer le scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_file_log">
          <property name="text">
           <string>üìã Logs d'ex√©cution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_file_log">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Les logs s'afficheront ici...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_file_report">
          <property name="text">
           <string>üìä Rapport de scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_file_report">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Le rapport d√©taill√© s'affichera ici...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_folder">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_folder_title">
          <property name="text">
           <string>üìÅ Scan de Dossier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_folder_selection">
          <layout class="QVBoxLayout">
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QPushButton" name="btn_select_folder">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>üìÇ Choisir un dossier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_folder_path">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Chemin du dossier...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_scan_folder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="text">
              <string>üîç Lancer le scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_folder_log">
          <property name="text">
           <string>üìã Logs d'ex√©cution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_folder_log">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_folder_report">
          <property name="text">
           <string>üìä Rapport de scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_folder_report">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_ip">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lbl_ip_title">
          <property name="text">
           <string>üåê Scan d'Adresse IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_ip_selection">
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLineEdit" name="input_ip">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Entrez une adresse IP (ex: 192.168.1.1 ou 8.8.8.8)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_scan_ip">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>45</height>
              </size>
             </property>
             <property name="text">
              <string>üîç Lancer le scan</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_ip_log">
          <property name="text">
           <string>üìã Logs d'ex√©cution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_ip_log">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_ip_report">
          <property name="text">
           <string>üìä Rapport de scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_ip_report">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>


